### ISSUE TITLE
Unresolved reference in working orbtk example

### ISSUE THREAD

**Issue by zee-mzha** [intellij-rust/intellij-rust#5943]
## Environment

* **IntelliJ Rust plugin version:** 0.3.128.3278-202
* **Rust toolchain version:** 1.45.2 (d3fb005a3 2020-07-31) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2020.2 (CL-202.6397.106)
* **Operating system:** Linux 5.8.0-1-default

## Problem description

Rust plugin complains about unresolved reference to `Window` even though window is properly included.

## Steps to reproduce

1. Add the latest version of orbtk to Cargo.toml (0.3.1-alpha3 as of time of writing)
2. Put the following into main.rs
```rust
use orbtk::prelude::*;

fn main(){
    Application::new()
        .window(|ctx|{
            Window::new()
                .build(ctx)
        })
        .run();
}
```

Expected behavior:

No issues should be reported

Current behavior:

"Window" in `Window::new()` should be red now with the error message "Unresolved reference: `Window` "

In both cases the program runs and compiles just fine but because of the above error there no code auto completion/suggestions.

`Window` is included through the prelude, which includes the widgets prelude

**Comment by dblanovschi** [comment]
I just ran into the same issue and tried digging a little into it.
It looks like the plugin either cannot find, cannot expand or cannot index the [`widget!` macro](https://github.com/redox-os/orbtk/blob/460029e3466988107969c5e4624e23200f403389/crates/api/src/macros.rs#L73-L538) that [`Window` is defined through](https://github.com/redox-os/orbtk/blob/460029e3466988107969c5e4624e23200f403389/crates/widgets/src/window.rs#L150-L177). A "Go To Declaration" on the widget macro where the Window is defined also results a "cannot find declaration to go to".

**Comment by qarmin** [comment]
Happens also with Intelij Idea 2020.2
Minimal project to import - 
[Minimal Project.zip](https://github.com/intellij-rust/intellij-rust/files/5177733/AA.zip)

**Comment by Undin** [comment]
Duplicate of #5146
